packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-java-visitor.test.ts

```diff
@@ -4,27 +4,27 @@ import { directives, scalars } from '../../scalars/supported-directives';
 import { AppSyncModelJavaVisitor } from '../../visitors/appsync-java-visitor';
 import { CodeGenGenerateEnum } from '../../visitors/appsync-visitor';
 import { JAVA_SCALAR_MAP } from '../../scalars';
-import { FeatureFlags } from 'amplify-cli-core';
-
-jest.mock("amplify-cli-core");
-const FeatureFlags_mock = FeatureFlags as jest.Mocked<typeof FeatureFlags>;
```

Comment:
Here and in other test files: Is the feature flag mock added as part of pipeline transformer related changes or otherwise, it might be used in other places to mock some of the features.

Reply:
I added the mock, this is into a feature branch and my previous PR added the mock, but Aaron showed me how codegen was handling feature flags so I switched my approach and deleted the old mocking related code